feeder/ethereum/README.md

```diff
@@ -0,0 +1,24 @@
+# Ethereum Witness Feeder
+
+## To run
+
+- cd into this directory (eg `feeder/ethereum`)
+- make sure your local environment matches the `.env.example` file in this directory
+
+### Continue with Docker
+
+- `docker run --rm -it $(docker build -q .)`
+  - This builds the container, runs it, and removes it after the run is complete.
+
+### Continue without Docker
+
+- make sure you have nodejs and yarn set up on your system
+- run `yarn` to install dependencies
+- run `yarn start` to run one iteration of the feeder
+- run `yarn dev` to run the feeder in "watch" mode (useful for development)
+
+## Explanation
```

Comment:
Maybe this section would be nice to have right at the top of this file to set a bit of background?

Reply:
Done.